public/static/docs/command-reference/import.md

```diff
@@ -73,11 +73,10 @@ data artifact from the source repo.
   an existing directory is specified, then the output will be placed inside of
   it.
 
-- `--rev` - specific
-  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
-  (such as a branch name, a tag, or a commit hash) of the repository to download
-  the file or directory from. The tip of the default branch is used by default
-  when this option is not specified.
+- `--rev` - commit hash, branch or tag name, etc. (any
+  [Git revision](https://git-scm.com/docs/revisions)) of the repository to
+  download the file or directory from. The latest commit in `master` (tip of the
```

Comment:
may be we can use `to import` here? instead of `to download`?

Reply:
We could. But I like that this is a copy paste of the same option in `dvc get` so that if there are future modifications we can just copy paste into the other doc. But again, lmk

Reply:
👍